docs/adr/0026-expand-subscription-platform-metadata.md

```diff
@@ -0,0 +1,81 @@
+# Expand Subscription Platform Metadata with Google Cloud Firestore
+
+- Status: proposed
+- Deciders: Ben Bangert, Barry Chen, Bianca Danforth
+- Date: 2021-06-24
+
+## Context and Problem Statement
+
+The Subscription Platform uses [Stripe Metadata][stripe-metadata] for numerous
+purposes. However, Stripe metadata has a limit on the number of values that we
+will reach in the near future. In order to continue to use metadata in the
+same fashion in the Subscription Platform, we need to overcome the limit placed
+by Stripe.
+
+[stripe-metadata]: https://stripe.com/docs/api/metadata
+
+## Decision Drivers
+
+- A higher storage limit than Stripe Metadata
+- A UI for the managers of products on the Subscription Platform to edit the
+  metadata
+- Access management for the UI
+- Updated data accessible in their respective environments without deployment
+
+## Considered Options
+
+- Google Cloud Firestore
+- MySQL
+- Configuration Files
+
+## Decision Outcome
+
+Chosen option: Google Cloud Firestore, because
+
+- Data management UI and user access control provided through the Google Cloud
+  Platform (GCP) console.
+- It does not require deployments for data updates.
+- It is not novel tech; FxA already uses it elsewhere.
+
+## Pros and Cons of the Options
+
+### Google Cloud Firestore
+
+[Google Cloud Firestore][cloud-firestore] is a hosted NoSQL document based database. It has access management and data management UI through the GCP console. We use it for the Event Broker.
+
+- Pros
+  - Already in the FxA monorepo; not a novel tech or a new dependency.
+  - Includes access management and data editing UI in the form of the GCP
+    console.
+  - Does not require deployment for data edits.
+- Cons
+  - Access control might not be granular enough.
```

Comment:
What do you mean here? Can you give an example or how it may not be granular enough?

Reply:
The ACL capability in Firestore actually looks more advanced and granular than MySQL: https://firebase.google.com/docs/rules

Reply:
I could remove that bullet point if the Firestore ACL rules meet our needs.

Reply:
I think if anything, the ACL capability in Firestore should be a Pro. It's fully programmatic as well, allowing functions to be used to gauge level of access on a per-field and per document basis for each action.